src/include/daos_fs.h

```diff
@@ -55,6 +55,7 @@ typedef struct {
 	daos_size_t		da_chunk_size;
 	/** Default Object Class for all objects in the container */
 	daos_oclass_id_t	da_oclass_id;
+	daos_prop_t		*da_props;
```

Comment:
we probably need a similar update to the duns struct to allow creating a POSIX container with checksum through the tool.

Reply:
yup

Reply:
done